tests/test_input.cpp

```diff
@@ -0,0 +1,41 @@
+#include <catch2/catch.hpp>
+
+#include <faasm/emulator.h>
+
+#include "faabric/util/func.h"
+#include "faasm/input.h"
+
+namespace tests {
+TEST_CASE("Test parse string to ints", "[input]")
+{
+    const char* input = "0 123 99987 5";
+    int* actual = faasm::parseStringToIntArray(input, 4);
+
+    REQUIRE(actual[0] == 0);
+    REQUIRE(actual[1] == 123);
+    REQUIRE(actual[2] == 99987);
+    REQUIRE(actual[3] == 5);
+
+    delete[] actual;
```

Comment:
Is this `delete[]` needed?

Reply:
Yes, because `parseStringToIntArray` calls `new`